### ISSUE TITLE
Deleted resource(s) appearing in description set

### ISSUE THREAD

**Issue by siuc-nate** [CredentialEngine/CredentialRegistry#416]
Fetching the description set for this framework:
https://credentialengineregistry.org/resources/ce-f93feb2a-1d1a-41da-92e2-3f77a53e20ac
![image](https://user-images.githubusercontent.com/21346519/111318985-8ce1e280-8633-11eb-84a6-7a2dfed0d8f4.png)

Yields a reference to the following two URIs, even though only the second one is actually referenced by the live record (and the first one does not exist - I assume it was deleted at some point):
https://credentialengineregistry.org/resources/ce-9d30f846-dfa4-4b1c-90fa-9d01238a86ac does not exist
https://credentialengineregistry.org/resources/ce-e423b7b7-90f6-47ae-8808-f476d1a8a7dd exists

![image](https://user-images.githubusercontent.com/21346519/111319079-a420d000-8633-11eb-9492-7fb815b193d2.png)

This framework also has competencies that reference other competencies that do not exist in the Registry:
![image](https://user-images.githubusercontent.com/21346519/111319395-f104a680-8633-11eb-8ca3-fa2aa56badd6.png)

The description set thus references those URIs as well - **however**, I think that is probably the correct/desired functionality, since the description set is merely reflecting what the actual record is asserting.

Thus the only real issue here, I think, is that the description set is not being updated to match the records in the Registry. 

In other words, the description set should only be referencing one URI for creator (and one for publisher), but should continue to list the invalid alignment URIs since those _also_ appear in the underlying competencies themselves.  I will discuss removing those alignments from those competencies with the CE team, and if we end up doing so, only then should they also be gone from the description set.

Hopefully that makes sense. It always gets a little confusing when two or more distinct problems intertwine like this.

**Comment by excelsior** [comment]
@siuc-nate Can confirm the first issue: description set URIs were not updated when respective resources got deleted. I resolved that and also fixed all of the existing records.

**Comment by siuc-nate** [comment]
Yes, the data has been corrected, thank you.  We will update the competencies to remove the references to missing data on our end (or possibly publish those competencies if they simply need to be published) and check back to make sure the description set data is updated accordingly.
